src/core/lib/iomgr/ev_epoll1_linux.cc

```diff
@@ -383,6 +383,14 @@ static void fd_shutdown_internal(grpc_fd* fd, grpc_error* why,
   if (fd->read_closure->SetShutdown(GRPC_ERROR_REF(why))) {
     if (!releasing_fd) {
       shutdown(fd->fd, SHUT_RDWR);
+    } else {
+      /* we need a dummy event for earlier linux versions. */
+      epoll_event dummy_event;
+      memset(&dummy_event, 0, sizeof(dummy_event));
```

Comment:
Do we need to this memset? It looks redundant.

Reply:
Not sure what exactly is meant by "ignored" in the doc and just to be safe. If you prefer, I can remove it and re-run some tests.

Reply:
I prefer to remove it.

Reply:
Cool. Done.